src/flip.jl

```diff
@@ -97,31 +104,37 @@ function flip_chain(graph::BaseGraph,
                     pop_constraint::PopulationConstraint,
                     cont_constraint::ContiguityConstraint,
                     num_steps::Int,
-                    score_keys::Array{String, 1},
+                    score_keys::Array{String, 1};

```

Comment:
is a `;` the customary way to seperate args with kwargs? just curious!

Reply:
yes! i was having a lot of trouble with this because without the semicolon the method literally would not run. the semicolon separates _keyword arguments_ from _positional arguments_!